rllib/utils/pre_checks.py

```diff
@@ -148,3 +158,57 @@ def get_type(var):
         done, bool), "Your step function must return a done that is a boolean."
     assert isinstance(
         info, dict), "Your step function must return a info that is a dict."
+
+
+def check_multiagent_environments(env: "MultiAgentEnv") -> None:
+    """Checking for common errors in RLlib MultiAgentEnvs.
+
+    Args:
+        env: The env to be checked.
+
+    """
+    from ray.rllib.env import MultiAgentEnv
+    if not isinstance(env, MultiAgentEnv):
+        raise ValueError("The passed env is not a MultiAgentEnv.")
+
+    reset_obs = env.reset()
+    sampled_obs = env.observation_space_sample()
```

Comment:
for multiagent env, both obs here are dicts right?
should we check that too?

Reply:
we can check that here, sgtm